android/expoview/src/main/java/versioned/host/exp/exponent/VersionedUtils.kt

```diff
@@ -56,27 +56,37 @@ object VersionedUtils {
   }
 
   private fun reloadExpoApp() {
-    val currentActivity = Exponent.instance.currentActivity
-    if (currentActivity is ReactNativeActivity) {
-      currentActivity.devSupportManager.callRecursive("reloadExpoApp")
-    } else {
+    val currentActivity = Exponent.instance.currentActivity as? ReactNativeActivity ?: return let {
       FLog.e(
         ReactConstants.TAG,
         "Unable to reload the app because the current activity could not be found."
       )
     }
+    val devSupportManager = currentActivity.devSupportManager ?: return let {
+      FLog.e(
+        ReactConstants.TAG,
+        "Unable to get the DevSupportManager from current activity."
+      )
+    }
```

Comment:
```suggestion
    val devSupportManager = currentActivity.devSupportManager ?: return run {
      FLog.e(
        ReactConstants.TAG,
        "Unable to get the DevSupportManager from current activity."
      )
    }
```
You don't have to change it to `run`, but for future uses, I think `let` isn't the best choice when we're talking about semantic.